common/mariadb-galera/helm/templates/statefulset-proxysql.yaml

```diff
@@ -107,6 +107,31 @@ spec:
         securityContext:
           privileged: {{ $.Values.initContainers.increaseMapCount.securityContext.privileged | default true }}
           runAsUser: {{ $.Values.initContainers.increaseMapCount.securityContext.runAsUser | default 0 | int }}
+      - name: fix-proxysql-fs-permissions
+        image: "{{ $.Values.image.os.registry }}/{{ $.Values.image.os.project }}/{{ $.Values.image.os.databasename }}:{{ $.Values.image.os.databaseversion }}-{{ $.Values.image.os.imageversion | int }}"
+        imagePullPolicy: {{ $.Values.image.os.pullPolicy | default "IfNotPresent" | quote }}
+        command:
+        - sh
+        - -c
+        - 'chown -R {{ $.Values.userId.proxy | default 3100 | int }}:{{ $.Values.groupId.proxy | default 3100 | int }} /mnt/*'
+        securityContext:
+          runAsUser: {{ $.Values.initContainers.fixProxysqlFsPermissions.securityContext.runAsUser | default 0 | int }}
```

Comment:
Please add a `initContainers.fixProxysqlFsPermissions.securityContext.runAsUser` option to the values.yaml similar to [cleanoscache](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/values.yaml#L856) and add some documentation similar to [wipeDataAndLog](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/values.yaml#L69-L71)

Reply:
done!